### ISSUE TITLE
Override alt-tab and alt-grave behaviour

### ISSUE THREAD

**Issue by Vaesper** [emasab/shelltile#24]
It'd be brilliant if any tiled window groups could behave as one "application" (one window group <==> one application) in gnome shell's alt-tab list and consequently be able to switch between windows in a window group using alt-grave. Not sure if it's possible to override the "application" of an existing window (and reset it when it is removed from the group), that would probably be the easiest implementation.

**Comment by AlexWayfer** [comment]
I don't agree. What if I want to change focus from one app in "group" to another?

**Comment by Vaesper** [comment]
You mean that inside the tiled group, you still want to have a distinction between apps and windows? I guess I generally don't tile so many windows that that is more useful to me than switching windows within the context of a "tiled group".
